src/internal/operations/restore.go

```diff
@@ -129,6 +130,12 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 
 	detailsStore := streamstore.New(op.kopia, op.account.ID(), op.Selectors.PathService())
 
+	ctx = clues.AddAll(
+		ctx,
+		"tenant_id", op.account.ID(), // TODO: pii
+		"backup_id", op.BackupID,
+		"service", op.Selectors.Service)
+
```

Comment:
Would it be possible.....to add a ticket for tech debt for this package for commons? There are times when the same `type` of functions in restore and backup.

Reply:
I very specifically want to avoid the use of `common` pkg here.  If we get so far as needing that, then it's a sign that the library itself has an unusable interface.  In which case, it'd be better to improve the clues package than to wrap it.  Ideally, the implementation should always involve a light touch, and not need any syntactical sugar.